src/main/java/com/google/step/finscholar/firebase/FirebaseStorageManager.java

```diff
@@ -96,7 +95,7 @@ public static void storeMultipleDocuments(Firestore database, String collectionT
     throws FirebaseException {
     for (Object object : objects) {
       try {
-        storeDocument(database, collectionToWriteTo, object, null);
+        storeDocument(database, collectionToWriteTo, object, Optional.ofNullable(null));
```

Comment:
Please reference the Optional documentation. This should be Optional.empty().

https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html#empty--

Reply:
Sg, thanks!